specification/advisor/resource-manager/readme.go.md

```diff
@@ -0,0 +1,46 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
```

Comment:
FYI -- in the `readme.go.md` having `$(go)`  guards is optional, since the file will only get loaded when --go is specified anyway. 

It's not harmful, it's just not necessary either.